### PR TITLE
DHFPROD-9697: Surface invalid model messages in mlLoadModules

### PR DESCRIPTION
**Pull request by ryanjdew**
#### Checklist: 
```diff
- Note: do not change the below
```

- ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
- [x] Ran cypress tests on firefox locally
  

- ##### Reviewer:

- [x] Reviewed Tests
- N/A Added to Release Wiki

### CODE REVIEW

**Review by aniamy — DISMISSED** [review]

**Review by prolonMarkLogic — COMMENTED** [review]
Hi @ryanjdew, I was testing this PR, but I noticed that the message error is the same,  I think the idea was to change it to be more descriptive right?

**Review by aniamy — DISMISSED** [review]

**Review by prolonMarkLogic — APPROVED** [review]

File: marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java [hunk 1]
```diff
@@ -101,9 +101,8 @@ public boolean saveQueryOptions() {
                 logger.info("Wrote entity-specific search options for Explorer to: " + stagingFile.getAbsolutePath());
                 FileUtils.writeStringToFile(expFinalFile, expOptions, Charset.defaultCharset());
                 logger.info("Wrote entity-specific search options for Explorer to: " + finalFile.getAbsolutePath());
+                return true;
```
Comment (aniamy) [inline marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java:104]: With this change we return false if only "options" were generated. Is that ok?

Comment (ryanjdew) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java:104]: Good catch. I'll adjust this to checkout that neither are null.

Comment (ryanjdew) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/impl/EntityManagerImpl.java]: I adjusted the logic to check that neither options were null.

**Review by aniamy — APPROVED** [review]

**Review by prolonMarkLogic — APPROVED** [review]

**Pull request merged**